board-games/test/player-test.js

```diff
@@ -0,0 +1,128 @@
+var assert = require('chai').assert;
+var Player = require('../src/player.js');
+var Game = require('../src/board-game.js');
+
+describe('Player', function() {
+  it.skip('should have a name', function() {
+    var player = new Player({name: 'Phil'});
+
+    assert.isFunction(Player);
+    assert.equal(player.name, 'Phil');
+  });
+
+  it.skip('should be able to have a favorite type of game', function() {
+    var player1 = new Player({name: 'Dave', favoriteType: 'Engine Builder'});
+    var player2 = new Player({name: 'Blair', favoriteType: 'Deck Builder'});
+
+    assert.equal(player1.favoriteType, 'Engine Builder');
+    assert.equal(player2.favoriteType,  'Deck Builder');
+  });
+
+  it.skip('should be able to have a collection', function() {
+    var game1 = new Game({name: 'Everdell'});
+    var game2 = new Game({name: 'Dominion'});
+    var player = new Player({name: 'Kiel', collection: [game1, game2]});
+
+    assert.equal(player.collection.length, 2);
+    assert.equal(player.collection[1].name, 'Dominion');
+  });
+
+  it.skip('should be able to add games to the collection', function() {
+    var game1 = new Game({name: 'Gloomhaven'});
+    var game2 = new Game({name: 'Spirit Island'});
+    var game3 = new Game({name: 'Scythe'});
+    var player = new Player({name: 'Zane', collection: []});
+
+    assert.deepEqual(player.collection, []);
+
+    player.addToCollection(game1);
+
+    assert.equal(player.collection.length, 1);
+    assert.equal(player.collection[0].name, 'Gloomhaven')
+
+    player.addToCollection(game2);
+    player.addToCollection(game3);
+
+    assert.equal(player.collection.length, 3);
+    assert.equal(player.collection[1].name, 'Spirit Island');
+    assert.equal(player.collection[2].name, 'Scythe');
+  });
+
+  it.skip('should be able to have a top 3 games', function() {
+    var game1 = new Game({name: 'Gaia Project'});
+    var game2 = new Game({name: 'Dune Imperium'});
+    var game3 = new Game({name: 'Terraforming Mars'});
+    var player = new Player({name: 'Phil'});
+
+    player.addToCollection(game1);
```

Comment:
Can you add one test before this one to make sure this.collection is assigned to an empty array by default?

Reply:
Done (lines 30-34)